circle.yml

```diff
@@ -20,3 +20,7 @@ test:
     override:
         - cd $GS_WD && go test -v $(glide novendor)
         - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
+        - cd $GS_WD &&
+          cp bindata.go bindata-repo.go &&
+          make update-bindata &&
+          cmp bindata.go bindata-repo.go
```

Comment:
Yes.

I can't think of a way to automate this. The only way I can think of is having a pre-commit hook that does that, but every developer needs to make sure the hook is there in their local repo clones.

Maybe somebody else has ideas how to do this better?